user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/storage_per_DMA/Profile_migration_to_elastic.md

```diff
@@ -0,0 +1,103 @@
+
+## Migrating from XML to the indexing database
+
+To migrate the profiles, you will need to use the SLNetClientTest tool. Note that while the tool mentions only "Elastic", it can also be used for a migration to OpenSearch.
+
+![SLNetClientTest tool](~/user-guide/images/ClientTestToolMigrationUI_ProfileMigration.jpg)<br>
+*Migration window in SLNetClientTest tool (version 10.4.2)*
+
+> [!WARNING]
+>
+> - Always be extremely careful when using this tool, as it can have far-reaching consequences on the functionality of your DataMiner System.
+> - All Profile Manager instances in the cluster will keep working during the migration. But any creates/updates done during the migration will no longer be migrated.
+> - After migration the storage type will still be XML, to switch to indexing database storage, press the *Start using Elastic* button.
+
+1. Connect to the DMS using the *SLNetClientTest* tool:
+
+    1. In the *SLNetClientTest* tool, go to *Connection* > *Connect*.
+    1. In the *Connect* window, under *Authentication*, select *Explicit credentials* and specify the credentials of an administrator user.
+    1. Click the *Connect* button.
+
+1. In the *Advanced* menu, select *Migration*.
+
+1. In the *ProfileManger XML to Elastic* section, click the *Start Migration* button to start the migration wizard.
+
+   During the migrations, the Profile Manager is not stopped, so it will keep working as expected. This also means a snapshot of the Profiles.xml is migrated. Any creates/updates done during the migration will no longer be migrated.
+
+1. A window will show the migration actions that have been scheduled. If you close this window, the migration will continue in the background. To cancel the migration, click the *Cancel Migration* button.
+
+   - The progress of the scheduled actions will be shown in the *MigrationStatus* table in SLNetClientTest tool, where a row will be created for the migration of profiles. These rows will be updated to reflect the progress of the migration.
+
+   - While the migration is in progress, a notice will be added in the Alarm Console. Once the migration is completed, the notice will be cleared and an information event will be generated, stating that the migration has finished.
+
+   - All profiles will be loaded from the *Profiles.xml* file on the DataMiner Agent where the migration was triggered. The *Profiles.xml* files of the other DataMiner Agents in the cluster are not migrated. This is not necessary, as the *Profiles.xml* file is synced in the cluster.
+
+   - The profiles objects will be written to the indexing database.
+
+   - When the migration for profiles is completed, the configuration will not switch to indexing database storage. To switch to indexing database storage, press the *Start using Elastic* button.
```

Comment:
As mentioned above, there might not be a config yet, in which case you first need to click the *Create Configuration* button